sdcm/scylla_bench_thread.py

```diff
@@ -141,6 +141,23 @@ def verify_results(self):
 
         return sb_summary, errors
 
+    def create_stress_cmd(self, stress_cmd):
+        if connection_bundle_file := self.node_list[0].parent_cluster.connection_bundle_file:
+            stress_cmd = f'{stress_cmd.strip()} -cloud-config-path={connection_bundle_file}'
+        else:
+            # Select first seed node to send the scylla-bench cmds
+            ips = ",".join([n.cql_ip_address for n in self.node_list])
+            stress_cmd = f'{stress_cmd.strip()} -nodes {ips}'
+
+        return stress_cmd
+
+    def connection_bundle_mount(self, loader):
+        mount_string = ''
+        if connection_bundle_file := self.node_list[0].parent_cluster.connection_bundle_file:
+            loader.remoter.send_files(connection_bundle_file, connection_bundle_file)
```

Comment:
You think giving it a hardcoded value would be better ?

Or a hardcoded directory/path ?

I rather keep the filename so the command would be clearer in the logs (in case of multiple tenants for example)

Reply:
Yes, hardcoded path like `/tmp/foo.bundle` will work just great.
Logs will contain unique loader name.